projects/rv/pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "ftrack-rv"
 # RV just accepts minor and major version
 # To match Backlight versioning we use year month patch (YY.MMP)
-version = "24.020"
+version = "24.20"
```

Comment:
This will not work for months like December, IMO we should use 24.020 and then in December 24.120